src/snap-confine.apparmor.in

```diff
@@ -21,6 +21,8 @@
     /usr/lib/@{multiarch}/libseccomp.so* mr,
     /lib/@{multiarch}/libseccomp.so* mr,
 
+    @{PROC}/@{pid}/auxv r,
+
```

Comment:
The debdiff I initially pasted did include this. If it is not required I will gladly remove it from this branch.